scipy/stats/stats.py

```diff
@@ -6099,9 +6104,13 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
     if a.size == 0 or b.size == 0:
         return _ttest_nans(a, b, axis, Ttest_indResult)
 
-    if permutations:
-        if int(permutations) != permutations or permutations < 0:
-            raise ValueError("Permutations must be a positive integer.")
+    if permutations is not None:
+        if permutations < 0 or (np.isfinite(permutations) and
+                                int(permutations) != permutations) :
```

Comment:
not sure how much we're concerned with typing here, do we need a check that `permutations` is of int type as well? Guessing not but asking to confirm.

Reply:
We could have cast it to an integer, but in the original PR we made it an error. I'm just maintaining that here.